.vscode/extensions.json

```
@@ -1,6 +1,11 @@
 {
     "recommendations": [
         "ms-dotnettools.csharp",
-        "editorconfig.editorconfig"
+        "editorconfig.editorconfig",
+        "ertanic.robust-lsp",
```

I'm not sure that we should be recommending robust-lsp over the stable Robust YAML. Robust LSP has around 5% of the installs, isn't directly affiliated/directly recommended by maintainers, and remotely installs an lsp binary.

I'd suggest using robust yaml instead as that is what the documentation suggests and it's been stable and is pretty actively used.

We could change over to suggesting the LSP, though I think we'd need maintainer approval.

```suggestion
        "slava0135.robust-yaml",
```